src/main/kotlin/org/rust/lang/refactoring/RsRenameProcessor.kt

```diff
@@ -0,0 +1,26 @@
+package org.rust.lang.refactoring
+
+import com.intellij.psi.PsiElement
+import com.intellij.psi.search.SearchScope
+import com.intellij.refactoring.rename.RenamePsiElementProcessor
+import org.rust.lang.core.psi.RsLabel
+import org.rust.lang.core.psi.RsLabelDecl
+import org.rust.lang.core.psi.RsLifetime
+import org.rust.lang.core.psi.RsLifetimeDecl
+
+class RsRenameProcessor : RenamePsiElementProcessor() {
+
+    override fun canProcessElement(element: PsiElement): Boolean = true
+
+    override fun prepareRenaming(element: PsiElement, newName: String?, allRenames: MutableMap<PsiElement, String>, scope: SearchScope) {
```

Comment:
I thought that too, but I see that Kotlin [allows nulls and has checks for that](https://github.com/JetBrains/kotlin/blob/3471b3311f8358a14ab21d957c6137a84615484a/idea/src/org/jetbrains/kotlin/idea/refactoring/rename/RenameKotlinPsiProcessor.kt#L61), so maybe there're some special cases? Though it seems unlikely, I'll remove it.